R/sort.R

```diff
@@ -0,0 +1,38 @@
+#' Automatically sort factor levels according to a user-defined function
```

Comment:
Maybe remove "Automatically"?

Reply:
The description now starts `#' Sort factor levels ...`